components/data-table/column.jsx

```diff
@@ -27,15 +27,15 @@ DataTableColumn.propTypes = {
 	 * import DataTableCell from 'design-system-react/data-table/cell';
 	 * const CustomDataTableCell = ({ children, ...props }) => (
 	 *   <DataTableCell {...props} >
-	 *     <a href="javascript:void(0);">{children}</a>
+	 *	 <a href="javascript:void(0);">{children}</a>
```

Comment:
The tabs in the comments should be reverted or they will end up as tabs in the code examples in the doc site.